src/main/java/first/controllers/StudentController.java

```diff
@@ -6,52 +6,50 @@
 import org.springframework.web.bind.annotation.*;
 
 @RestController
-public class StudentController implements StudentControllerInter {
+public class StudentController implements BaseHtmlTemplate {
+    StudentRepository studentRepository = new StudentRepository();
+    Student student;
```

Comment:
This is bad idea to use Student student as class variable because of concurrency